README.md

```diff
@@ -53,12 +53,12 @@ Jenkins is **licensed** under the **[MIT License]**. The terms of the license ar
     OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
     THE SOFTWARE.
 
-[ButlerImage]: http://jenkins-ci.org/sites/default/files/jenkins_logo.png
+[ButlerImage]: https://jenkins.io/sites/default/files/jenkins_logo.png
 [MIT License]: https://github.com/jenkinsci/jenkins/raw/master/LICENSE.txt
 [Mirrors]: http://mirrors.jenkins-ci.org
 [GitHub]: https://github.com/jenkinsci/jenkins
 [website]: http://jenkins-ci.org
```

Comment:
The new website is https://jenkins.io/ . All http://jenkins-ci.org are outdated. It seems to be aligned with your HTTP => HTTPS fixes